README.md

```diff
@@ -1,25 +1,41 @@
-Metrics server is responsible for collecting resource metrics from kubelets and exposing them in Kubernetes Apiserver
-through [Metrics API]. Main consumers of those metrics are `kubectl top`, [HPA] and [VPA]. Metric server stores only the
-latest values of metrics needed for core metrics pipeline (CPU, Memory) and is not responsible for forwarding metrics
-to third-party destinations.
+Metrics Server is a simple, scalable, resource efficient source of container resource metrics needed for Kubernetes
+built-in autoscaling pipelines.
 
-## Design
+Metrics Server collects resource metrics from Kubelets and expose them in Kubernetes apiserver through [Metrics API] to
+be used by [Horizontal Pod Autoscaler] and [Vertical Pod Autoscaler]. Metrics API can also be accessed by `kubectl top`
+allowing for easier debugging of autoscaling pipelines.
 
-The detailed design of the project can be found in the following docs:
+Metric Server is not meant for non-autoscaling purposes, it is not responsible for forwarding metrics to monitoring
+solutions nor should be used as source of metrics for them.
 
-- [Metrics API design]
-- [Metrics Server design]
+Metrics Server good points:
+- Simple - single deployment that should out-of-the box on most clusters (see [Requirements](#requirements) section)
+- Scalable - supports up to 5k node clusters
+- Resource efficient - uses 0.5m core of CPU and 4 MB of memory per node
+
+## Use cases
+
+Metrics Server **can** be used when you want:
+- CPU/Memory based horizontal autoscaling (learn more about [Horizontal Pod Autoscaler])
+- Automatic adjusting/suggesting resources needed by containers (learn more about [Vertical Pod Autoscaler])
+
+Metrics Server **should not** be used when you need:
+- Non-Kubernetes clusters
+- Accurate source of resource usage metrics
+- Horizontal autoscaling based on other resources then CPU/Memory
+
+For unsupported use cases please checkout full monitoring solutions like Prometheus.
 
 ## Requirements
 
-Metrics server has particular requirements on cluster and network configuration that is not default for all cluster distributions. Please ensure that your cluster distribution supports those requirements before using metrics server:
-* Metrics server needs to be reachable from kube-apiserver ([Configuring master to cluster communication])
-* Kube-apiserver should be correctly configured to enable aggregation layer ([How to configure aggregation layer])
-* Nodes need to have kubelet authorization configured and match metrics-server configuration ([How to configure kubelet authorization])
-* Pod/Node metrics need to be exposed by Kubelet by Summary API
+Metrics server has particular requirements on cluster and network configuration that is not default for all cluster
+distributions. Please ensure that your cluster distribution supports those requirements before using Metrics Server:
+- Metrics server needs to be reachable from kube-apiserver ([Configuring master to cluster communication])
```

Comment:
It's better to link in-line. You'll need to add an additional link for `[reachable from kube-apiserver]` at the end of the file.

```suggestion
- Metrics Server must be [reachable from kube-apiserver]
```